### PR TITLE
Models tests

### PR DESCRIPTION
**Pull request by jorbush**

### CODE REVIEW

File: backend/models/plan.py [hunk 1]
```diff
@@ -176,11 +176,16 @@ def add_participant(self, uuid: str):
             raise PlanDBAddingError() from e
 
     def remove_participant(self, uuid: str):
-        if uuid not in self._participants:
+        if uuid not in [p.uuid if isinstance(p, User) else p for p in self._participants]:
```
Comment (AlbertMartinezM) [inline backend/models/plan.py:179]: nice to point this out! If i am not wrong you could directly use self.get_plan_participants_id(). I should have done this, my bad.

File: backend/models/user.py [hunk 2]
```diff
@@ -26,14 +25,14 @@ def json(self) -> dict:
...
+        add new user into the db. It generates the user uid.
         :return: None
         """
         try:
-            update_time, us_ref = db.collection(u'users').add({u'username': self.username, u'joined': self.joined})
+            update_time, us_ref = db.collection(u'users').add({u'username': self.username, u'joined': self.joined}, timeout=90)
             self.uuid = us_ref.id
         except Exception as e:
-            raise UserDBAddingError(self) from e
+            raise e
```
Comment (AlbertMartinezM) [inline backend/models/user.py:35]: would be greate to keep the exception UserDBAddingError() from e. ( I understand that you have temporally changed it in order to see what is really going on)

File: backend/models/user.py [hunk 3]
```diff
@@ -57,4 +56,14 @@ def get_user(cls, uuid: str):
         try:
             return User.from_dict(u)
         except Exception as e:
-            raise UserCreationError(u) from e
\ No newline at end of file
+            raise UserCreationError(u) from e
+
+    def delete_user(self):
```
Comment (AlbertMartinezM) [inline backend/models/user.py:61]: Great! I understand that it is just for testing purposes but keep in mind that user uuid is also stored in plan model and maybe would be great to delete there too?

File: backend/tests/unit/test_plan.py
Comment (AlbertMartinezM) [inline backend/tests/unit/test_plan.py:1]: Great Job! Seems nice

File: backend/tests/unit/test_user.py
Comment (AlbertMartinezM) [inline backend/tests/unit/test_user.py:1]: 💯 👍

**Review by AlbertMartinezM — CHANGES_REQUESTED** [review]
Check the comments 😺

File: backend/models/user.py (hunk 2 above)
Comment (jorbush) [reply backend/models/user.py]: I've fixed. Nice catch!

File: backend/models/plan.py (hunk 1 above)
Comment (jorbush) [reply backend/models/plan.py]: You are right, I have fixed. Thank u

File: backend/models/user.py (hunk 3 above)
Comment (jorbush) [reply backend/models/user.py:61]: All the data is removed when the tests finished, so I am not going to do that implementation for "testing". I am sorry

Comment (AlbertMartinezM) [reply backend/models/user.py:61]: sure, no problem 😄

**Review by AlbertMartinezM — APPROVED** [review]
Everything seems fine!

**Pull request merged**